lib/validate_encrypt_utils.pm

```diff
@@ -80,19 +170,39 @@ sub verify_cryptsetup_properties {
     }
 }
 
+=head2 verify_restoring_luks_backups
+
+  verify_restoring_luks_backups(%args);
+
+Where C<%args> expects the following parameters:
+=over
+
+=item C<$mapped_device>
+  path of encrypted device
+
+=item C<$backup_file_info>
+   unique string to match with the info of the backup file
+
+=item  C<$backup_path_path>
```

Comment:
Nitpicking warning: same as above, two spaces instead of one.

If you don't mind :)

Reply:
done